compair/configuration.py

```diff
@@ -90,4 +90,9 @@
     if os.environ.get(env) != None:
         config[env] = strtobool(os.environ.get(env))
 
-# print config
\ No newline at end of file
+
+# force cas and lti login methods to not be enabled when is demo installation
+if config['DEMO_INSTALLATION'] == True:
+    config['APP_LOGIN_ENABLED'] = True
+    config['CAS_LOGIN_ENABLED'] = False
+    config['LTI_LOGIN_ENABLED'] = False
```

Comment:
Remove the force setting of `LTI_LOGIN_ENABLED` (can be set to whatever is desired in demos)